sensor/common/sensor/sensor.go

```diff
@@ -89,6 +91,8 @@ func NewSensor(configHandler config.Handler, detector detector.Detector, imageSe
 		currentStateMtx: &sync.Mutex{},
 
 		stoppedSig: concurrency.NewErrorSignal(),
+
+		notifyList: []common.Notifiable{},
```

Comment:
forgot to mention before, but no need to initialize this, as the first `append` will take care of that

Reply:
Done